torch/cuda/nvtx.py

```diff
@@ -9,21 +11,40 @@ def _fail(*args, **kwargs):
         rangePushA = _fail
         rangePop = _fail
         markA = _fail
+        rangePushEx = _fail
+        markEx = _fail
+        version = None
+        size = None
+
 
     _nvtx = _NVTXStub()  # type: ignore[assignment]
 
 __all__ = ['range_push', 'range_pop', 'mark']
 
+#TODO: Should these be pulled in from nvToolsExt.h directly?
+colors = {
+        'blue':        0x003498db,
+        'green':       0x002ecc71,
+        'yellow':      0x00f1c40f,
+        'orange':      0x00e67e22,
+        'red':         0x00e74c3c,
+        'purple':      0x009b59b6,
+        'navy':        0x0034495e,
+        'gray':        0x0095a5a6,
+        'silver':      0x00bdc3c7,
+        'darkgray':    0x007f8c8d,
+        }
 
-def range_push(msg):
+def range_push(msg,color=colors['silver']):
```

Comment:
IS the default color really silver? If the default is "no color" it would be better to express it directly this way